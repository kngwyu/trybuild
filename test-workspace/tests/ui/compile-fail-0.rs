use std::rc::Rc;

struct NotSend {
    value: Rc<usize>,
}

impl test_crate::Sendable for NotSend {}

fn main() {}
