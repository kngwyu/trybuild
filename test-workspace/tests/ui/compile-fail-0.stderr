error[E0277]: `std::rc::Rc<usize>` cannot be sent between threads safely
 --> $DIR/compile-fail-0.rs:7:6
  |
7 | impl test_crate::Sendable for NotSend {}
  |      ^^^^^^^^^^^^^^^^^^^^ `std::rc::Rc<usize>` cannot be sent between threads safely
  |
 ::: D:\\a\\trybuild\\trybuild\\test-workspace\\src\\lib.rs:1:21
  |
1 | pub trait Sendable: Send {}
  |                     ---- required by this bound in `test_crate::Sendable`
  |
  = help: within `NotSend`, the trait `std::marker::Send` is not implemented for `std::rc::Rc<usize>`
  = note: required because it appears within the type `NotSend`
